backend/src/types.ts

```diff
@@ -555,3 +555,16 @@ export type PersistentVolumeClaimListKind = {
   metadata: Record<string, unknown>;
   items: PersistentVolumeClaimKind[];
 };
+
+export type PrometheusResponse = {
+  data: {
+    result: [
+      {
+        metric: any;
```

Comment:
we can remove the `metric` for now if we don't need it in our code to avoid `any` type.

Reply:
ok, removed.